sdk/core/core-paging/types/corePaging.d.ts

```diff
@@ -14,10 +14,10 @@ export interface PageSettings {
   maxPageSize?: number;
 }
 /**
-* @interface
-* An interface that allows async iterable iteration both to completion and by page.
-*/
-export interface PagedAsyncIterableIterator<T, PageT = never> {
+ * @interface
+ * An interface that allows async iterable iteration both to completion and by page.
+ */
+export interface PagedAsyncIterableIterator<T, PageT = never, PageSettingsT = PageSettings> {
```

Comment:
Maybe it's the C# talking, but wouldn't these be `TPage` and `TPageSettings`?

Reply:
I have no horse in this race. @bterlson  ?

Reply:
Eh since nobody really looks too hard at parameter names for generics, I am going to merge this as-is.